sklearn/multioutput.py

```diff
@@ -150,3 +156,134 @@ def score(self, X, y, sample_weight=None):
         from .metrics import r2_score
         return r2_score(y, self.predict(X), sample_weight=sample_weight,
                         multioutput='uniform_average')
+
+
+def _check_estimator(estimator):
+    """Make sure that an estimator implements the necessary methods."""
+    if (not hasattr(estimator, "fit") and not hasattr(estimator, "score") and
+            not hasattr(estimator, "predict_proba")):
+        raise ValueError("The base estimator should implement fit,"
```

Comment:
Why should the base estimator implement `predict_proba` if I simply want to `fit`?